Backend/app/rider-platform/rider-app/Main/src/API.hs

```diff
@@ -97,5 +100,7 @@ juspayWebhookHandler ::
   BasicAuthData ->
   Value ->
   FlowHandler AckResponse
-juspayWebhookHandler merchantShortId secret =
-  withFlowHandlerAPI . Payment.juspayWebhookHandler merchantShortId secret
+juspayWebhookHandler merchantShortId secret val = withFlowHandlerAPI $ do
```

Comment:
here also we have to change , other wise we won't we able to remove  city dependency from the merchant table , like for the dashboard part , lets create a city table also , so we will be having three tables merchant table , city, and merchantOperating city , so now instead of passing the merchant id we can pass. the composite key of the city and merchant id.